src/shared/services/assignment-labels-service/assignment-labels-service.gql.ts

```diff
@@ -47,15 +47,12 @@ export const UPDATE_ASSIGNMENT_LABEL = gql`
 `;
 
 export const DELETE_ASSIGNMENT_LABELS = gql`
-	mutation deleteAssignmentLabel($profileId: uuid!, $labelIds: [Int!]!) {
-		delete_app_assignment_labels(
+	mutation deleteAssignmentLabel($profileId: uuid!, $labelIds: [uuid!]!) {
+		delete_app_assignment_labels_v2(
```

Comment:
i'm guessing this also deletes the link entries in the `assignments_v2_assignment_labels_v2` table through a cascade on delete?